pkg/config/template.go

```diff
@@ -1105,27 +1105,31 @@ const templateStringCrioRuntimeLogFilter = `# Filter the log messages by the pro
 const templateStringCrioRuntimeUIDMappings = `# The UID mappings for the user namespace of each container. A range is
 # specified in the form containerUID:HostUID:Size. Multiple ranges must be
 # separated by comma.
+# This option is deprecated, and will be replaced with native Kubernetes user namespace support in the future.
```

Comment:
Do we want to include a link to the k8s.io docs here?

Reply:
hm how about adding the KEP number instead?: